core/src/main/javascript/reportviewer/reportviewer.js

```diff
@@ -1189,7 +1189,7 @@ define([ 'common-ui/util/util', 'common-ui/util/timeutil', 'common-ui/util/forma
                     hideDlgAndPane(registry.byId('feedbackScreen'));
 
                     //Show loading screen
-                    $('#notification-message').html(_Messages.getString('LoadingPage') + " " + mainJobStatus.page + " " + _Messages.getString('Of') + " " + mainJobStatus.totalPages);
+                    $('#notification-message').html(xssUtil.sanitizeHtml(_Messages.getString('LoadingPage') + " " + mainJobStatus.page + " " + _Messages.getString('Of') + " " + mainJobStatus.totalPages));
```

Comment:
Should use the `xssUtil.setHtml(..)` method, after support for jQuery objects has been added to `setHtml(..)` and `setHtmlUnsafe(..)`.

Reply:
See https://github.com/pentaho/pentaho-platform-plugin-reporting/pull/864#discussion_r1831504782.

Reply:
Updated